docs/roadmap.md

```diff
@@ -1,85 +1,7 @@
-Here is the roadmap for **Pydantic Logfire**. This is a living document, and it will be updated as we progress.
+This [Github issue][roadmap] is the

```

Comment:
It doesn't feel worth keeping this whole page and a top level item in the sidebar when the page is now so small. If we want we could draw attention to the roadmap by pinning it to the github issues. Maybe we could link to it from the help page or something.

Reply:
doesn't really hurt to have it for searching etc.